zipline/finance/slippage.py

```diff
@@ -128,6 +132,25 @@ def to_dict(self):
         del py['type']
         return py
 
+    def __getstate__(self):
+
+        state_dict = self.__dict__
+
+        STATE_VERSION = 1
+        state_dict[VERSION_LABEL] = STATE_VERSION
+
+        return state_dict
+
+    def __setstate__(self, state):
+
+        OLDEST_SUPPORTED_STATE = 1
+        version = state.pop(VERSION_LABEL)
+
+        if version < OLDEST_SUPPORTED_STATE:
+            raise BaseException("Transaction saved state is too old.")
```

Comment:
can we refactor this logic out into a check_versino function along with a custom error? I don't mind the BaseException but joe will yell at me if I don't ask for a custom exception :/